tests/manager.test.js

```diff
@@ -0,0 +1,31 @@
+const Manager = require("../lib/manager.js")
+
+// test to see if the manager constructor is working correctly
+describe("Manager", () => {
+    it("should create a manager object", () => {
+       const manager = new Manager ("John", 123, "john@email.com", "Birmingham University");
```

Comment:
We don't need to create this manager for every test. The code is just repeated and is unecessary. If we move this out to the `describe` block, so after line 4, we can reuse the same `manager` for each test block.